webpack.static.config.babel.js

```diff
@@ -48,18 +48,21 @@ const locals = {};
 function getStaticSitePaths() {
   const glob = require('glob');
 
+  const lessonSrcPath = lessonSrc.replace(/\\/g, '/');
```

Comment:
Skriver du en kommentar?
F.eks. "Need to replace backslashes with forward slashes on Windows, since glob keeps forward slashes".